tests/integrationv2/apache2/nix/conf/apache2.conf

```diff
@@ -0,0 +1,254 @@
+KeepAlive On
+MaxKeepAliveRequests 100
+KeepAliveTimeout 5
+HostnameLookups Off
+ErrorLog ${APACHE_LOG_DIR}/error.log
+LogLevel info
+
+# Inline load module
+LoadModule access_compat_module ${APACHE_NIX_STORE}/modules/mod_access_compat.so
+LoadModule alias_module ${APACHE_NIX_STORE}/modules/mod_alias.so
+LoadModule auth_basic_module ${APACHE_NIX_STORE}/modules/mod_auth_basic.so
+LoadModule authn_core_module ${APACHE_NIX_STORE}/modules/mod_authn_core.so
+LoadModule authn_file_module ${APACHE_NIX_STORE}/modules/mod_authn_file.so
+LoadModule authz_core_module ${APACHE_NIX_STORE}/modules/mod_authz_core.so
+LoadModule authz_host_module ${APACHE_NIX_STORE}/modules/mod_authz_host.so
+LoadModule authz_user_module ${APACHE_NIX_STORE}/modules/mod_authz_user.so
+LoadModule autoindex_module ${APACHE_NIX_STORE}/modules/mod_autoindex.so
+LoadModule dir_module ${APACHE_NIX_STORE}/modules/mod_dir.so
+LoadModule deflate_module ${APACHE_NIX_STORE}/modules/mod_deflate.so
+LoadModule env_module ${APACHE_NIX_STORE}/modules/mod_env.so
+LoadModule filter_module ${APACHE_NIX_STORE}/modules/mod_filter.so
+LoadModule log_config_module ${APACHE_NIX_STORE}/modules/mod_log_config.so
+LoadModule mime_module ${APACHE_NIX_STORE}/modules/mod_mime.so
+LoadModule mpm_event_module ${APACHE_NIX_STORE}/modules/mod_mpm_event.so
+LoadModule reqtimeout_module ${APACHE_NIX_STORE}/modules/mod_reqtimeout.so
+LoadModule rewrite_module ${APACHE_NIX_STORE}/modules/mod_rewrite.so
+LoadModule setenvif_module ${APACHE_NIX_STORE}/modules/mod_setenvif.so
+LoadModule socache_shmcb_module ${APACHE_NIX_STORE}/modules/mod_socache_shmcb.so
+LoadModule ssl_module ${APACHE_NIX_STORE}/modules/mod_ssl.so
+LoadModule status_module ${APACHE_NIX_STORE}/modules/mod_status.so
+LoadModule unixd_module ${APACHE_NIX_STORE}/modules/mod_unixd.so
+
+# Include list of ports to listen on
+Include conf/ports.conf
+
+<IfModule unixd_module>
+  User ${APACHE_RUN_USER}
+  Group ${APACHE_RUN_GROUP}
+</IfModule>
+
+DocumentRoot ${APACHE_SERVER_ROOT}/www/html
+<Directory />
+	Options FollowSymLinks
+	AllowOverride None
+	Require all denied
+</Directory>
+
+<Directory ${APACHE_SERVER_ROOT}/www/html>
+       Options Indexes FollowSymLinks
+       AllowOverride None
+       Require all granted
+</Directory>
+
+AccessFileName .htaccess
+
+LogFormat "%v:%p %h %l %u %t \"%r\" %>s  \"%{Referer}i\" \"%{User-Agent}i\"" vhost_combined
+LogFormat "%h %l %u %t \"%r\" %>s  \"%{Referer}i\" \"%{User-Agent}i\"" combined
+LogFormat "%h %l %u %t \"%r\" %>s " common
+LogFormat "%{Referer}i -> %U" referer
+LogFormat "%{User-agent}i" agent
+CustomLog ${APACHE_LOG_DIR}/other_vhosts_access.log vhost_combined
+
+# Include the virtual host configurations:
+IncludeOptional sites-enabled/*.conf
+
+# inline module configs
+<IfModule mod_mime.c>
+	TypesConfig ${APACHE_NIX_STORE}/conf/mime.types
+	AddType application/x-compress .Z
+	AddType application/x-gzip .gz .tgz
+	AddType application/x-bzip2 .bz2
+	AddLanguage am .amh
+	AddLanguage ar .ara
```

Comment:
Are all of these necessary?

Reply:
We can likely get rid of some of the languages, but not sure on the dependencies on charsets.

Reply:
I'd suggest removing what you can for simplicity. Unless this file was copied exactly from somewhere, and modifying it makes this trickier?

Reply:
I've removed most of the extra languages and any unrecognizable charsets; apache starts fine and tests pass locally.